include/frozen/string.h

```diff
@@ -86,6 +86,10 @@ class basic_string {
     return size() < other.size();
   }
 
+  friend constexpr bool operator>(const basic_string& lhs, const basic_string& rhs) {
+    return rhs < lhs;
```

Comment:
I tilted on "why isn't that `lhs > rhs` ?

Reply:
That would be infinite self-recursion.